defender/tests.py

```diff
@@ -491,49 +491,50 @@ def test_is_valid_ip(self):
     def test_parse_redis_url(self):
```

Comment:
Can we add one test with a password that needs to be quoted and we pass in True to parse_redis_url for password_quote. And another test for a password that doesn't need a quote but also uses the password_quote=True, and make sure the results are what we expect?

Reply:
finished
PTAL, Thanks

Reply:
Looks Good, thank you.